1-exercises/D-do-while/exercise.js

```diff
@@ -8,6 +8,17 @@
 
 function evenNumbersSum(n) {
```

Comment:
Please remember about indentation and spacing in javascript:
https://courses.cs.washington.edu/courses/cse154/17au/styleguide/js/spacing-indentation-js.html#:~:text=Spacing%20and%20indentation%20should%20be,than%20the%20previous%20line%27s%20indentation.